### PR TITLE
XW-3251 | tabs components in Design System

### PR DESCRIPTION
**Pull request by weronika**
https://wikia-inc.atlassian.net/browse/XW-3251

### CODE REVIEW

File: app/templates/route-components/tabs.hbs [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+<p>
+	Tabs are used for handling content in a more ordered way and also may be used to build complex navigation solutions.
+</p>
+
+<section>
+	<h2 id='text-only'>Tabs with text only</h2>
+	<p>
+		To display a set of tabs you need to place them inside <code>&lt;ul&gt;</code> tag with class <code class='format-code'>.wds-tabs-container</code>
+		Each <code>&lt;li&gt;</code> item (tab) needs to have <code class='format-code'>.wds-tab</code> class and some link <code>&lt;a&gt;</code> or plain text
+		<code>&lt;span&gt;</code> inside with class <code class='format-code'>.wds-tab-label</code>.
```
Comment (rogatty) [inline app/templates/route-components/tabs.hbs]: What do you think about having these?
- `.wds-tabs`
- `.wds-tabs__tab`
- `.wds-tabs__tab-label`?

That would follow what we have in e.g. `/identity/global-footer`:
```
<footer class="wds-global-footer">
  <h2 class="wds-global-footer__header">	
    <svg class="wds-global-footer__header-logo">
```

There is no single tab without the tabs container, so all the elements together are basically a single component, right? The naming should reflect that.

File: app/templates/route-components/tabs.hbs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	</p>
+
+	{{#component-demo}}
+		{{#wds-tabs-container}}
+			{{#wds-tab}}
+				<span class='wds-tab-label'>Text tab</span>
+			{{/wds-tab}}
+
+			{{#wds-tab}}
+				<a href='#'  class='wds-tab-label'>A really long tab that breaks into lines and is a link</a>
```
Comment (latata) [inline app/templates/route-components/tabs.hbs]: Two spaces between attributes ;)

File: style-guide/styles/wds-components/_tabs.scss [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+	$tab-padding-horizontal: 12px;
+	$tab-padding-vertical: 13px;
+	$tab-default-color: $wds-color-link;
+	$tab-hover-color: darken($tab-default-color, 20%);
+	$tab-icon-size: 14px;
+	$tab-chevron-size: 10px;
+	$tab-icon-margin: 6px;
+
+	cursor: pointer;
+	line-height: 1.17;
```
Comment (latata) [inline style-guide/styles/wds-components/_tabs.scss]: `1.17`? maybe we can change it to `14px` as specified in designs?

File: style-guide/styles/wds-components/_tabs.scss [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+			padding-left: $tab-padding-horizontal + $tab-icon-size + $tab-icon-margin;
+
+			.wds-icon {
+				height: $tab-icon-size;
+				left: $tab-padding-horizontal;
+				margin-top: -$tab-icon-size / 2;
+				min-width: $tab-icon-size;
+				position: absolute;
+				top: 50%;
+				vertical-align: top;
```
Comment (latata) [inline style-guide/styles/wds-components/_tabs.scss]: do we need it?

File: dist/scss/wds-components/_tabs.scss [hunk 5]
```diff
@@ -0,0 +1,113 @@
+.wds-tabs {
+	align-items: flex-end;
+	display: flex;
+	list-style: none;
+	padding: 0;
+}
+
+.wds-tabs__tab {
+	$tab-padding-horizontal: 12px;
```
Comment (hakubo) [inline dist/scss/wds-components/_tabs.scss]: nice use of local variables

File: app/templates/route-components/tabs.hbs [hunk 6]
```diff
@@ -0,0 +1,121 @@
+<h1>Tabs</h1>
+<p>
+	Tabs are used for handling content in an ordered way and also may be used to build complex navigation solutions.
+</p>
+
+<section>
+	<h2 id='text-only'>Tabs with text only</h2>
```
Comment (rogatty) [inline app/templates/route-components/tabs.hbs]: https://github.com/Wikia/guidelines/tree/master/Handlebars#quotes
>HTML attribute values should be wrapped in double quotes. Strings inside handlebars helpers should use single quotes.

**Pull request merged**